src/dftbp/dftb/rangeseparated.F90

```diff
@@ -6,1629 +6,6250 @@
 !--------------------------------------------------------------------------------------------------!
 
 #:include 'common.fypp'
+#:include 'error.fypp'
```

Comment:
I did not yet do anything about this, you resolved it intentionally?

Reply:
The new code in getddHfNumericalGammaDeriv doesn't match the intention of the macro, as it uses a local error() call inside the function, instead of passing the messenger variable upwards (and also calls the propogate macro, which is broken as this will just return the function in this case). These message variables are supposed to intent(out) all of the way up to the level of main.F90, where it then gets delt with.

Reply:
> I did not yet do anything about this, you resolved it intentionally?
Nope, I didn't do it either.

Reply:
Hm, my GUI says `bhourahine marked this conversation as resolved.`, seems to be a buggy. Let me unresolve it then.

Reply:
I had another discussion with Bálint and what is implemented right (in particular rshGamma.F90 and rangeseparated.F90) now essentially reflects the outcome of that.